src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  let totalSum;
+
+  if (days >= 7) {
+    totalSum = rentPerDay * days - 50;
+  } else if (days >= 3 && days < 7) {
+    totalSum = rentPerDay * days - 20;
+  } else {
+    totalSum = rentPerDay * days;
+  };
```

Comment:
```suggestion
  if (days > 6) {
    return (rentPerDay * days) - 50;
  }
  
  if (days > 2) {
    return (rentPerDay * days) - 20;
  }
  
  return totalSum * days;
```
Если правильно прописать условия - можно упростить конструкцию

Reply:
Повыноси числа в переменные тоже(3, 7, 20, 50)